pages/en/release-notes/graphql-validations-migration-guide.mdx

```diff
@@ -0,0 +1,536 @@
+---
+title: GraphQL Validations migration guide
+---
+
+Soon `graph-node` will support 100% coverage of the [GraphQL Validations specification](https://spec.graphql.org/June2018/#sec-Validation).
+
+Previous versions of `graph-node` did not support all validations and provided more graceful responses - so, in cases of ambiguity, `graph-node` was ignoring invalid GraphQL operations components.
+
+GraphQL Validations support is the pillar for the upcoming new features and the performance at scale of The Graph Network.
+
+It will also ensure determinism of query responses, a key requirement on The Graph Network.
+
+**Enabling the GraphQL Validations will break some existing queries** sent to The Graph API.
+
+To be compliant with those validations, please follow the migration guide.
+
+> ⚠️ If you do not migrate your queries before the validations are rolled out, they will return errors and possibly break your frontends/clients.
+
+# Migration guide
```

Comment:
I think here it should make clear that these are two separate approaches - either use the CLI, or you can just swap out the URL in your frontend and test it